lib/ansible/module_utils/facts/system/pkg_mgr.py

```diff
@@ -13,7 +13,7 @@
 # A list of dicts.  If there is a platform with more than one
 # package manager, put the preferred one last.  If there is an
 # ansible module, use that as the value for the 'name' key.
-PKG_MGRS = [{'path': '/usr/bin/rpm-ostree', 'name': 'atomic_container'},
+PKG_MGRS = [{'path': '/usr/bin/rpm-ostree', 'name': 'rpm_ostree_pkg'},
```

Comment:
Do we want to maintain the backward compatibility where `atomic_container` is package manger.

Reply:
It seems like we'd need to use more information than just the presence of rpm-ostree, and we are definitely concerned about the change in behavior here...

Reply:
This list is in order of precedence and this is the least preferred option IIUC. I honestly doubt this `rpm-ostree` entry in the list was ever hit. Below in the code (where I switched `atomic_container` to `rpm_ostree_pkg`) it detects if `/run/ostree-booted` on RH systems and will use `rpm_ostree_pkg` instead of `atomic_container`. I suspect that's where `atomic_container` won the heuristic lottery here if ever.